lib/moleculer/service/base.rb

```diff
@@ -74,8 +85,33 @@ def events
           @events ||= {}
         end
 
+        ##
+        # returns the full name of the service, including version and prefix
+        # @return string
+        def full_name
+          return @full_name if @full_name
+
+          unless @version
+            @full_name = service_name
+            return @full_name
+          end
+
+          name    = service_name.dup
+          version = @version.to_s
+          version.prepend("v") if @version.is_a? Numeric
+
+          if name.include? "."
```

Comment:
```suggestion
          if name.include?(".")
```

for code consistency